modules/ROOT/pages/database-administration/syntax.adoc

```diff
@@ -186,10 +262,14 @@ DROP [COMPOSITE] DATABASE name [IF EXISTS] [RESTRICT \| CASCADE ALIAS[ES]] [{DUM
 [[administration-syntax-database-alias-management]]
 == Database alias management command syntax
 
-[options="header", width="100%", cols="1,5a"]
+The following tables cover both local and remote database aliases.
```

Comment:
My understanding is that it is only the very next table which cover both, after that it is separate tables for local and remote since there are some variations. So I think you either want to write just "The following table..." instead of "The following tables..." or clarify it a bit.

Reply:
I think it's the general `tables` as in all of these tables together covers both? but that sentence came from @renetapopova  so I'll let her decide if we want to update it

Reply:
No I do not think removing 'both' helps. I interpret this sentence as each of the following tables (unclear how many) covers both local and remote alias, which is not true. Some tables cover both, some cover local and some cover remote if I understand the layout of the page correctly

Reply:
Maybe it is just me that interpret the sentence incorrectly, if you think it is correct already I am fine with leaving it. Same for my other comment which is about the same thing but in another place

Reply:
```suggestion
The following sections cover the commands related to local and remote database aliases.
```
is that better?

Reply:
or feel free to come up with your own suggestion

Reply:
the sentence isn't supposed to talk about just the next one so changing to `The following table` isn't the way to go